cmd/lhsm-plugin-az/main.go

```diff
@@ -60,6 +60,7 @@ type (
 		Bandwidth             int        `hcl:"bandwidth"`
 		MountRoot             string     `hcl:"mountroot"`
 		ExportPrefix          string     `hcl:"exportprefix"`
+		AzureEnvironment      string     `hcl:"az_env"`
```

Comment:
Hmmm I feel like we are losing some flexibility here if we tie the config to well known values. What if the cx's storage account has a custom domain?